employees/views.py

```diff
@@ -229,8 +231,10 @@ def get_context_data(self, **kwargs: Any) -> dict:
                 "daily_hours_sum": self.daily_hours_sum,
                 "monthly_hours_sum": self.monthly_hours_sum,
                 "UI_text": ReportListStrings,
+                "UI_filter_text": FilterOptionLabels,
                 "project_form": self._create_project_join_form(),
                 "hide_join": self.hide_join,
+                "style": self.style,
```

Comment:
I don't really understand why something related with style is here, but it will disappear with rest views.

Reply:
Done